Gopkg.toml

```diff
@@ -72,3 +76,7 @@ required = [
 [[constraint]]
   name = "github.com/onsi/ginkgo"
   version = "1.5.0"
+
+[[override]]
+  name = "github.com/gophercloud/gophercloud"
```

Comment:
🤦‍♂️ https://github.com/kubernetes/kops/issues/5551

Reply:
Great stuff. I hadn't got around to making kops changes eyt (just using as is). The kops PR makes sense. Perhaps i'll have a look after i get teh final changes on this done.